backend-forms.md

```diff
@@ -363,6 +364,18 @@ Radio lists can also support a secondary description.
 
 Radio lists support three ways of defining the options, exactly like the [dropdown field type](#field-dropdown). For radio lists the method could return either the simple array: **key => value** or an array of arrays for providing the descriptions: **key => [label, description]**
 
+<a name="field-balloon"></a>
+### Balloon Selector
+
+`balloon-selector` - renders a list, where only one item can be selected at a time.
```

Comment:
I think that `renders a list of button options, where only one item can be selected at a time.` from #146 is a slightly better description as well, so if you could add that it'd be great.